docs/local_install.md

```diff
@@ -14,7 +14,7 @@ This guide will help you to get Substra source code and to start a local instanc
 
 - docker:
   - on macOS, by default, docker has access only to the user directory.
-    - Substra requires access to `/substra`, update accordingly the docker desktop configuration (`Preferences` > `File Sharing`).
+    - Substra requires access to `/tmp/substra`, update accordingly the docker desktop configuration (`Preferences` > `File Sharing`).
```

Comment:
I think the path depends on the value of the env var `SUBSTRA_PATH` (at least for the setup with docker-compose).
Could be good to add a bit more information here.

Reply:
Done!